### DISCUSSION TITLE
docker/docker_compose_volumes_establish_kafka/

### DISCUSSION THREAD

**Discussion by giscus** [ambosing/ambosing.github.io#12] [Blog-Comments]
# docker/docker_compose_volumes_establish_kafka/

Docker!!!!!!!!!!! 너무 편해요..!

https://ambosing.github.io/docker/docker_compose_volumes_establish_kafka/

**Comment by yunjae228** [comment]
포스팅 잘 읽었습니다 !! 궁금한 게 있는데 컴포즈에서 root권한을 주면 따로 인증을 요구하진 않나요 ??
만약 compose up 시에 ./logs 디렉토리가 없으면 어떻게 되나요 ??

**Reply by ambosing** [reply]
docker compose에서 따로 인증을 요구하지 않고 위 포스팅처럼 바로 에러가 나버립니다..!
그래서 사실 에러 찾기가 조금 더 어려웠어요. 직관적이지 않아서
./logs 폴더 없으면 에러가 발생합니다! 
파일이 불일치는 될 때는 도커쪽에서 파일을 가져오더라구요! (저도 처음 앎)
